test/plugins/glossary/definition/handle-backspace-in-term.js

```diff
@@ -0,0 +1,37 @@
+/** @jsx h */
+
+export default (change, editor) => {
+    editor.run('onKeyDown', { key: 'Backspace' })
+}
+
+export const input = <value>
+    <document>
+        <p>Text</p>
+        <definition>
```

Comment:
Glossary can only contain definitions.

Reply:
Hmm, ok. I will have to change how empty glossary behaves.